cli/src/lib/git.js

```diff
@@ -25,6 +25,19 @@ async function getGitPath() {
   }
 };
 
+export async function getDiff() {
+  const gitPath = await getGitPath();
+  try {
+    const {stdout} = await child_process.spawnP(
+      gitPath,
+      ['diff', '--name-only', 'HEAD~1'],
```

Comment:
This only checks the diff of the last commit. If we want it to check what differs from origin/master we can get it by 

```shell
git diff --name-only $(git merge-base origin/master HEAD)
```